hub/instruction_handling/kec.tex

```diff
@@ -117,57 +117,13 @@ \subsubsection{Constraints}
 				exoSum            = \exoWeightKec         ,
 				phase             = \nothing              ,
 				}
-			% \setMmuInstructionParametersRamToExoWithPadding {i}{\miscRowOffset}
-			% \left[ \begin{array}{ll}
-			% 	\utt{Source ID:}            & \cn_{i}               \\
-			% 	\utt{Target ID:}            & \nothing              \\
-			% 	\utt{Auxiliary ID:}         & 1 + \hubStamp_{i}     \\
-			% 	% \utt{Source offset high:} & \col{src\_offset\_hi} \\
-			% 	\utt{Source offset low:}    & \locOffsetLo          \\
-			% 	% \utt{Target offset low:}  & \col{tgt\_offset\_lo} \\
-			% 	\utt{Size:}                 & \locSizeLo            \\
-			% 	% \utt{Reference offset:}   & \col{ref\_offset}     \\
-			% 	\utt{Reference size:}       & \locSizeLo            \\
-			% 	\utt{Success bit:}          & \nothing              \\
-			% 	% \utt{Limb one:}           & \col{limb\_1}         \\
-			% 	% \utt{Limb two:}           & \col{limb\_2}         \\
-			% 	\utt{Exo sum:}              & \exoWeightKec         \\
-			% 	\utt{Phase:}                & \nothing              \\
-			% \end{array} \right]
 		\]
-		% \[
-		% 	\left\{ \begin{array}{lclr}
-		% 		\miscMmuFlag                  _{i + \miscRowOffset}  & = & \one  & (\trash) \vspace{2mm} \\
-		% 		\miscMmuInst                  _{i + \miscRowOffset} & = & \locInst \\
-		% 		\miscMmuInstParamOne          _{i + \miscRowOffset} & = & \hashInfoStamp_{i} \\
-		% 		\miscMmuInstParamTwo          _{i + \miscRowOffset} & = & 0 & (\trash) \\
-		% 		% \miscMmuReturner              _{i + \miscRowOffset} & = & 0 & (\trash) \\
-		% 		\miscMmuInfo                  _{i + \miscRowOffset} & = & 0 \\
-		% 		\miscMmuRefOffset             _{i + \miscRowOffset} & = & 0 \\
-		% 		\miscMmuRefSize               _{i + \miscRowOffset} & = & 0 \\
-		% 		\miscMmuOffsetOneLo           _{i + \miscRowOffset} & = & \locOffsetLo \\
-		% 		\miscMmuOffsetTwoHi           _{i + \miscRowOffset} & = & 0 & (\trash) \\
-		% 		\miscMmuOffsetTwoLo           _{i + \miscRowOffset} & = & 0 & (\trash) \\
-		% 		\miscMmuSize                  _{i + \miscRowOffset} & = & \locSizeLo \\
-		% 		\miscMmuStackValHi            _{i + \miscRowOffset} & = & 0 & (\trash) \\
-		% 		\miscMmuStackValLo            _{i + \miscRowOffset} & = & 0 & (\trash) \\
-		% 		\miscMmuExoSum                _{i + \miscRowOffset} & = & \ob{TODO!} \\
-		% 	\end{array} \right.
-		% \]
-		\saNote{} The current context and caller context (which may own the call data) are provided by the lookup. 
-
-		\saNote{} We are providing extra information to the \mmuMod{} in case of a \inst{CALLDATALOAD}:
-		(\emph{a})
-		the knowledge whether the current execution context is the root context
-		(\emph{b})
-		the absolute transaction number.
-		The purpose of the former is to be able to retrieve the \textbf{transaction call data} whenever we are executing a \inst{CALLDATALOAD} in the root context of a transaction. 
```

Comment:
This paragraph was more than likely a copy paste error from the `COPY` instruction family.